docs/getting-started/enterprise/key-connector.mdx

```diff
@@ -47,6 +50,42 @@ libssl was found".
 4. If you are running the Key Connector from a terminal, restart your terminal to make sure the
    updated `.zshrc` settings are applied
 
+</TabItem>
+<TabItem value="arm" label="ARM">
+
+Given that the Key Connector project is based on NET 5 then we need to use the x86_64 version of
+OpenSSL, thus installing x86_64 packages using Homebrew (a guide with several approaches can be
+found
+[here](https://www.wisdomgeek.com/development/installing-intel-based-packages-using-homebrew-on-the-m1-mac/)).
+
+1. Install Rosetta
+   ```bash
+   /usr/sbin/softwareupdate --install-rosetta
```

Comment:
❓ could this be simplified to just `softwareupdate --install-rosetta` 
That's what I ran to install it.